services/finishedQuiz.js

```diff
@@ -0,0 +1,12 @@
+const FinishedQuiz = require('~/models/finishedQuiz')
+
+const finishedQuizService = {
+  getFinishedQuizzes: async (skip = 0, limit = 10) => {
+    const items = await FinishedQuiz.find().skip(skip).limit(limit).sort({ createdAt: -1 }).lean().exec()
+    const count = await FinishedQuiz.countDocuments()
+
```

Comment:
We should return finished quizzes only for a specific author, not all.

Reply:
changed